### ISSUE TITLE
pre-commit: gitlab.com/PyCQA/flake8 no longer exists

### ISSUE THREAD

**Issue by carmenbianca** [OCA/oca-addons-repo-template#173]
## Module

Any Odoo 14 module, probably more. To test, try https://github.com/OCA/cooperative @ 14.0.

## Describe the bug

When running pre-commit, you get something like this:

```
$ pre-commit run --all
[INFO] Initializing environment for https://gitlab.com/PyCQA/flake8.
Username for 'https://gitlab.com': 
```

This is because the pre-commit configuration contains this:

```yaml
  - repo: https://gitlab.com/PyCQA/flake8
    rev: 3.8.3
    [...]
```

And https://gitlab.com/PyCQA/flake8 leads to a 404. I think this is recent?

## To Reproduce
**Affected versions**: Odoo 14, probably more.

1. Clear pre-commit cache.
2. Run `pre-commit run --all-files`

**Expected behavior**

flake8 downloads properly.

**Additional context**
Add any other context about the problem here. (e.g. OS, Python version, ...)

Not relevant.

**Comment by pedrobaeza** [comment]
It was fixed in #170. Please run `pipx run copier -f update` on the repo to update to latest definitions.

**Comment by carmenbianca** [comment]
Ah, thank you @pedrobaeza !
